ci/build_wheel_python.sh

```diff
@@ -24,9 +24,13 @@ cd "${package_dir}"
 echo "libkvikio-${RAPIDS_PY_CUDA_SUFFIX} @ file://$(echo ${CPP_WHEELHOUSE}/libkvikio_*.whl)" > ./constraints.txt
 
 PIP_CONSTRAINT="${PWD}/constraints.txt" \
+SKBUILD_CMAKE_ARGS="-DUSE_CUDA_NVCOMP_WHEEL=ON" \
```

Comment:
Should this just be `USE_NVCOMP_WHEEL`? (no CUDA?)

Reply:
I've updated the name.